### PR TITLE
Updated README

### LINKED ISSUE #19: Update Readme
**Describe the bug**
Readme currently has some old links to systers, and some other proprietary links, find them all and change them accordingly.
Reduce image size in readme as it takes up a lot of space.
**Expected behavior**
Readme should be updated with latest links to resources and information

### PR DESCRIPTION
**Pull request by bartekpacia**
### Description
I have:
- updated the README file with a few useful links
- added Code Style section to CONTRIBUTING file
- bumped some dependencies to their newest possible versions.
- removed selfish information about how I started this project from the README file :)

Fixes #19 

### Flutter Channel:
- [x] I have used the Flutter Beta channel on my local machine 

### Type of Change:
- Code
- Documentation

### How Has This Been Tested?
Ran locally on my Android emulator (Pixel 3a API 29). Everything works fine.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,21 +36,24 @@ Almost everything.
 - RelationPage: updating `availableToMentor` and `needsMentoring` status doesn't work
```
Comment (isabelcosta) [inline README.md:36]: these 2 should be removed and each bullet turned into issues

File: README.md [hunk 2]
```diff
@@ -34,21 +36,24 @@ Almost everything.
...
 - About screen is basically About dialog. It's very simple, more info should be added.
 
-Occasional UI bugs may occur – in this case, please create an issue.
+Occasional UI bugs may occur – in this case, please [file an issue](https://github.com/anitab-org/mentorship-flutter/issues/new/choose).
 
 ### Overview
 
-- App _tries_ to follow Clean Architecure guidelines. Logic is separated into 4 layers:
+The architecture of the app isn't top-notch, but currently it does it job quite well. If you have an idea
```
Comment (isabelcosta) [inline README.md:43]: I think this sentence is not needed, its a bit self deprecating. For example, what is top notch architecture!? There is always room for improvement in a project. Backend and Android aren't perfect and could have some refactoring, and that is fine :)
Let's keep this objective. if someone wishes to open an issue to improve the app, they'll do it anyways :) If you feel something is clearly missing you can even create the issue yourself, for someone to solve.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,23 +1,36 @@
 # Contributing Guidelines
```
Comment (isabelcosta) [inline CONTRIBUTING.md:1]: Can you do the change to the CONTRIBUTING.md on another PR, you are mixing a lot of changes not related with the issue you are trying to solve, which is just about the README and not contributing guidelines. You can create an issue to improve the contributing guidelines and submit a PR for it. To help with this, its useful to create a branch per change you do, to be able to have parallel PRs with different commits associated.

File: README.md [hunk 4]
```diff
@@ -68,14 +73,6 @@ To use web version of this app with your local server:
    to enable CORS
 2. Run local devserver
 
-### Future?
```
Comment (isabelcosta) [inline README.md:71]: well removed :)

File: pubspec.lock [hunk 5]
```diff
@@ -42,7 +42,7 @@ packages:
       name: bloc
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "3.0.0"
```
Comment (isabelcosta) [inline pubspec.lock:45]: can you also do these bumps in a separate PR, since this is not related to the README.

File: README.md [hunk 6]
```diff
@@ -34,21 +36,24 @@ Almost everything.
 - RelationPage: updating `availableToMentor` and `needsMentoring` status doesn't work
 - About screen is basically About dialog. It's very simple, more info should be added.
 
-Occasional UI bugs may occur – in this case, please create an issue.
+Occasional UI bugs may occur – in this case, please [file an issue](https://github.com/anitab-org/mentorship-flutter/issues/new/choose).
```
Comment (isabelcosta) [inline README.md:39]: Maybe you can add a Contributing section here and include a text to highlight the CONTRIBUTING guidelines that should be read before starting to contribute. Also the reporting guidelines and code of conduct

File: README.md [hunk 7]
```diff
@@ -1,18 +1,20 @@
 # Mentorship Client
 
-Cross-platform client for [Systers&#x27; mentorship system](https://github.com/systers/mentorship-backend),
-written in [Flutter](https://flutter.dev/).
+Cross-platform client for the [AnitaB.org's](https://anitab.org/) [Mentorship System](https://github.com/anitab-org/mentorship-backend),
```
Comment (isabelcosta) [inline README.md]:
```suggestion
[Mentorship System](https://github.com/anitab-org/mentorship-backend) is an application that allows women in tech to mentor each other, on career development topics, through 1:1 relations for a certain period. This is the flutter client for this project.
```
I would avoid having AnitaB.org's because there are other platforms that AnitaB.org uses, and this is still in development and part of the open source community for now.

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@bartekpacia the changes to the README was in my mind for some time so I am glad you took initiative to solve it ;)
I left some notes which summarize in these 2 points:
- in this community, we try to have the PR's code focused on the issue they are solving, and since the commits are squashed in the end, it can be misleading and difficult to track, changes that are not related, being done within the same commit, such as README improvements and bumps in the dependencies versions
- you did very good changes to the README, I just added some notes about the wording and a missing section that can help members understand the importance of following our contribution guidelines

File: README.md (hunk 2 above)
Comment (bartekpacia) [reply README.md:43]: Yeah, this makes sense :)
I'll change it.

File: README.md (hunk 4 above)
Comment (bartekpacia) [reply README.md:71]: ^^

File: pubspec.lock (hunk 5 above)
Comment (bartekpacia) [reply pubspec.lock:45]: can I say "next time"? ;)
It's a really small bump

**Comment by techno-disaster** [comment]
can you make the image a bit smaller/ align it to the right too? it feels too big/long and takes up a lot of space in the Readme

File: CONTRIBUTING.md (hunk 3 above)
Comment (bartekpacia) [reply CONTRIBUTING.md:1]: I completely agree with you Isabel about too much stuff in this PR. But what would you say if  I (only in this PR ;) ) just changed PR title to "update README and CONTRIBUTING"? 
I understand the strive for best practices but...just this small PR 🙄

Comment (bartekpacia) [reply CONTRIBUTING.md:1]: I must sound unprofessional now hahah

**Comment by bartekpacia** [comment]
@Techno-Disaster sure, will check this soon. I just copied them from some other PR so that's probably why they're broken.

File: pubspec.lock (hunk 5 above)
Comment (isabelcosta) [reply pubspec.lock:45]: Can I respond with 🤦 ? i'm thinking...

File: CONTRIBUTING.md (hunk 3 above)
Comment (isabelcosta) [reply CONTRIBUTING.md:1]: > I understand the strive for best practices but...just this small PR

The "but" in that sentence... 🤣 I mean, do you want to follow best practices or nah?
Do you want to inspire the contributors to follow best practises, or will you be fine with them saying "just this small PR"?

Comment (isabelcosta) [reply CONTRIBUTING.md:1]: If ... when we reach a level that this is deployed on production, you want to uphold these standards to your contributors who devote time to help you build the app, so that we minimize confusion in the commit history, we have small Pull Requests to easily review and test, have consistency in the way we develop the project and common guidelines for everyone to follow, etc...

Comment (techno-disaster) [reply CONTRIBUTING.md:1]: Oh no. 🤦😂 i agree with isabel now.
Also i don't think we need a PR  to bump the numbers specifically, as this is a small update let contributors bump the numbers when they update the flutter code.Maybe we can add a
[ ] have you updated the packages to their latest versions from pub.dev field? in the PR template?
What are your thoughts

**Comment by bartekpacia** [comment]
Closing and dividing into 3 commits.

🤦‍♂️
